Cargo.toml

```diff
@@ -22,30 +22,42 @@ travis-ci = { repository = "crossbeam-rs/crossbeam" }
 [features]
 default = ["std"]
```

Comment:
Should the `default` set of features also contain `"alloc"`?

Reply:
I think it is not necessary because this PR enables Items requiring `alloc` under the following conditions.

https://github.com/crossbeam-rs/crossbeam/blob/61e15de65fcb70f82dbade006efef857885032d9/crossbeam-skiplist/src/lib.rs#L26

Reply:
In addition to this, this PR does `extern crate alloc` when `alloc` feature is enabled (The minimum required version of the `alloc` feature is Rust 1.36, so it is newer than the minimum required version of crossbeam).

We can change this to prioritize the std feature, but it is not recommended to include `alloc` as the default, as it may add dependencies that have similar implementations (that breaks the minimum required version).

Reply:
Makes sense, thanks!